### DISCUSSION TITLE
Problem with image uploading

### DISCUSSION THREAD

**Discussion by necropaulis** [community/community#133591] [New to GitHub]
I'm trying to do a QR Code card.

I had it displaying everything properly, but realized it wasn't displaying correctly in mobile (problem related with the area the card was in, nothing to do with the image).
I went to fix the space issue, and then the image went away-the only thing I messed with was margin (404ed)
I went into Inspect, found the image is now trying to pull from local, instead of Github.

Things I've tried:
All manners of /, ./, ../, etc
Direct link github/user/qr/img
Removing repository, and posting it under a slightly different name
1 & 2 again.
Checking from my phone to make sure it's actually on Github

When I change code on Github, it changes when I go to VSCode, and the other way around.

While I'm new to all this, I've already done a contact card, and it works perfectly fine.

Anyone ever see this??

**Comment by KuyaMecky** [comment]
It sounds like you're dealing with a broken image path issue that's manifesting on mobile devices after adjusting margins. Here are a few troubleshooting steps to help resolve the issue:

1. **Check the Image Path**: Make sure the image path you’re using is correct and accessible. Verify that the path you’re using for the image matches the one on GitHub. If the image is in a repository, the URL should follow the format:
   ```
   https://raw.githubusercontent.com/username/repository/branch/path/to/image
   ```

2. **Inspect Element**: Use your browser’s developer tools (Inspect Element) to see the exact URL being requested for the image. This can help confirm if the path is correct or if it’s being altered somehow.

3. **Verify Accessibility**: Open the direct image URL in a browser to make sure it’s accessible and correctly loading. If you get a 404 error, then the path might be incorrect or the file might not be in the expected location.

4. **Clear Caches**: Sometimes browsers cache old versions of files. Clear your browser’s cache and refresh the page to ensure you're seeing the latest version of your code.

5. **Responsive Design**: Ensure that the image is not being hidden or repositioned incorrectly due to CSS media queries or responsive design settings. Check if the CSS rules for mobile view are affecting the image visibility.

6. **Relative vs. Absolute Paths**: If you’re using relative paths (e.g., `./img/qr.png`), try switching to absolute paths to ensure there’s no ambiguity in locating the image.

7. **Check for Errors**: Look for any errors in the console that might indicate problems loading the image or other related issues.

If you still have trouble, providing the relevant code snippet or URL could help diagnose the issue more specifically.

**Comment by necropaulis** [comment]
It was the cache, and using the autosave in VSCode.
